server/game/cards/01_SOR/units/SuperlaserTechnician.ts

```diff
@@ -0,0 +1,24 @@
+import AbilityHelper from '../../../AbilityHelper';
+import { NonLeaderUnitCard } from '../../../core/card/NonLeaderUnitCard';
+import { Location } from '../../../core/Constants';
+
+export default class SuperlaserTechnician extends NonLeaderUnitCard {
+    protected override getImplementationId() {
+        return {
+            id: '8954587682',
+            internalName: 'superlaser-technician'
+        };
+    }
+
+    public override setupCardAbilities() {
+        this.addWhenDefeatedAbility({
+            title: 'Put Superlaser Technician into play as a resource and ready it',
+            optional: true,
+            immediateEffect: AbilityHelper.immediateEffects.sequential([
+                AbilityHelper.immediateEffects.moveCard((context) => ({ target: context.source, destination: Location.Resource })),
```

Comment:
I'm realizing that we probably need a MoveCardToResourceSystem to handle rules around resourcing cards consistently, and in case there's ever anything that triggers off of it. Fine to merge this for now but can you add a TODO here and then we can hold off on adding any more cards that resource themselves (or other cards) until that's in place.